<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCountries</class>
 <widget class="QMainWindow" name="ShowCountries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>223</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>311</width>
    <height>223</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>311</width>
    <height>223</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vis innlagte land</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jobber.png</normaloff>jobber.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLandID">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>102</y>
      <width>44</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Navn*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLandID">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>102</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCountryName">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>135</y>
      <width>77</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Landnavn*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCountryName">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>135</y>
      <width>142</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>170</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Slett</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>275</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string>Her kan du se de ulike statusene som er registrert i systemet. Vil du endre på ID-en til en status, må det gjøres i kommandolinja. Alle felt må utfylles.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
